### DISCUSSION TITLE
New user here: npm version tags are misleading

### DISCUSSION THREAD

**Discussion by Acumane** [yarnpkg/berry#5300] [Q&A] [answered]
When installing from npm,

- `@berry` still on 2.x. Why? I expected 3.x, which isn't even available as a release on [npm](https://www.npmjs.com/package/yarn?activeTab=versions)
- I would've expected 1.x to be `@classic` (not `@latest`) as per its rebranding
- Why not `@1`, `@2`, and `@3` to dispel all this ambiguity?

I'm privy to some of the challenges faced when releasing berry, but as a new user these naming conventions have been fairly confusing/misleading. Could someone let me in on the reasoning?

**Comment by milesj** [comment, marked as answer]
@Acumane Yarn 2/3 is installed _in the repository_ ([example](https://github.com/moonrepo/moon/tree/master/.yarn/releases)), and is not managed via npm. However, the v1 binary on npm will hook into the Yarn installation at `.yarn/releases` and execute with that.

I'll agree, it is a bit confusing.
